### DISCUSSION TITLE
VIF file

### DISCUSSION THREAD

**Discussion by flomo29** [seaweedfs/seaweedfs#2723] [Q&A]
Why is there a file called .vif ?
What is the purpose of that file?

**Comment by chrislusf** [comment]
it is meant to be volume information. It is used as volume metadata, such as a volume remote location.

**Reply by flomo29** [reply]
So when i upload a volume .dat file to the cloud tier with s3 and i delete the vif file, the volume server doesn´t know anything about the uploaded .dat file? Or is the information stored in the .idx file too? Does that mean, when i have only the .vif file and not the .idx file and i call ./weed fix ... does weed download the .dat file to recreate the .idx file?
